code/modules/assembly/flash.dm

```diff
@@ -97,6 +100,12 @@
 
 /obj/item/assembly/flash/handheld/weak
 	bulb = /obj/item/flashbulb/weak
+	name = "cyborg eye"
+	desc = "A small but powerful light with a shielded camera lens embedded in the center. Cyborgs need these to see what they're doing. The camera is non-functional until it is attached to a cyborg head"
+	icon = 'icons/obj/machines/camera.dmi'
+	icon_state = "cameracase"
+	lefthand_file = null
+	righthand_file = null
```

Comment:
handheld/weak is for weak flashes, not cyborg eyes specifically. Make a new subtype for cyborg eyes.

Reply:
There are six weak flashes located on every map in the robotics lab at roundstart. My understanding is these are explicitly for making cyborgs with. Outside of these six, the only way to get more is by printing them. 

They are not implemented elsewhere at all. Creating a new subtype is completely pointless because I would be replacing these six spawns *and* printed version with cyborg eyes leaving no references to the weak flash

Reply:
Same with above. Just because its not currently used elsewhere, doesn't mean it should have a misleading typepath